src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml

```diff
@@ -110,7 +110,7 @@
                                            FontStyle="Italic"
                                            Text="="/>
 
-                                <TextBox x:Name="ValueTextBox"
```

Comment:
This will break the variable functionality as this name is used in EquationInputArea::SubmitTextbox. Here and the rest of the removed x:Names in this file.